docs/source/environments/braintumor-1d.rst

```diff
@@ -0,0 +1,269 @@
+.. _braintumor:
+
+
+Brain Tumor 1D PDE
+================
+
+This documentation describes the 1D Brain Tumor DPR (Diffusion-Proliferation-Radiation) Environment. The environment models the growth dynamics of glioblastomas–a fast-growing type of brain cancer–and their response to external beam radiation therapy (XRT). The model is governed by a partial differential equation (PDE) that includes three key terms: diffusion of tumor cells, proliferation, and radiation-induced cell death.
+
+Background
+------------------------
+
+Glioblastoma is the most common malignant primary brain tumor, accounting for approximately 16% of all primary brain tumors [3]_.
```

Comment:
this should be [1] not [3]. Reorder your references to be in the order of which you use them.